pandas/_libs/tslib.pyx

```diff
@@ -83,6 +84,12 @@ def ints_to_pydatetime(const int64_t[:] arr, object tz=None, object freq=None,
          convert to this timezone
     freq : str/Offset, default None
          freq to convert
+    fold : bint, default is 0
+        Due to daylight saving time, one wall clock time can occur twice
+        when shifting from summer to winter time; fold describes whether the
+        datetime-like corresponds  to the first (0) or the second time (1)
+        the wall clock hits the ambiguous time
+        .. versionadded:: 1.1.0
```

Comment:
i think you need a blank line before versionadded

Reply:
Done.